packages/thumbprint-react/components/Image/index.jsx

```diff
@@ -61,7 +61,7 @@ const Image = forwardRef((props, outerRef) => {
     // React Waypoint and lazysizes use. The React Waypoint README explains this concept well:
     // https://git.io/fj00H
 
-    const parent = canUseDOM && scrollparent(containerRef);
+    const parent = canUseDOM && containerRef && scrollparent(containerRef);
```

Comment:
Was this an existing bug you found during this change?

Reply:
The third-party scrollparent library returned undefined when it received null/undefined nodes. This one does not, but I prefer this behavior.